### ISSUE TITLE
Add packaging metadata

### ISSUE THREAD

**Issue by Alexander-Wilms** [Realm667/WolfenDoom#143]
Freedoom has .desktop and [.appdata.xml](https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html#sect-Quickstart-DesktopApps) files in their repository, which makes packaging it easier: https://github.com/freedoom/freedoom/tree/master/dist

Thanks to this, it looks quite nice in Linux package managers: https://beta.flathub.org/apps/details/io.github.Freedoom-Phase-1

Could you add such files as well? Examples:

```
[Desktop Entry]
Name=WolfenDoom: Blade of Agony
Comment=Story-driven FPS inspired by WWII shooters from the 90's and early 2000's
GenericName=First Person Shooter Game
Type=Application
Icon=m_doom
Keywords=first;person;shooter;doom;
Categories=Game;ActionGame;
Exec=gzdoom
```

The <metadata_license> field should be one of
* FSFAP
* CC0-1.0
* CC-BY-3.0
* CC-BY-SA-3.0
* GFDL-1.3
* MIT 
so it can be combined with arbitrary other data in one file that contains all the metadata on all applications in a repository.
```
<?xml version="1.0" encoding="UTF-8"?>
<component type="desktop">
  <id>com.realm667.boa.desktop</id>
  <name>WolfenDoom: Blade of Agony</name>
  <summary>Story-driven FPS inspired by WWII shooters from the 90's and early 2000's</summary>
  <metadata_license>CC0-1.0</metadata_license>
  <project_license>???</project_license>
  <developer_name>Realm667</developer_name>
  <url type="homepage">http://boa.realm667.com</url>
  <url type="bugtracker">https://github.com/Realm667/WolfenDoom/issues</url>
  <description>
  <p>
    Blade of Agony is a story-driven FPS. The project is inspired by WWII shooters from the 90's and early 2000's, like Wolfenstein 3D, Medal of Honor, and Call of Duty, but with faster-paced gameplay in the spirit of Doom! The game can be played standalone using the GZDoom engine as a base.
  </p>
  <p>
    STORY

    It's 1942, and the war approaches its peak. The U.S. has joined the Allied forces, and the Soviets are pushing back the front in the east. The tides of war are turning, and Hitler's victory seems more and more beyond his grasp. But the Nazis, refusing to capitulate, have grown obsessed with human experiments and occult artifacts, in which they see a potential escape from their final Doom. The Allied leaders reject this possibility as nonsense; nonetheless, some still fear what the Führer might be up to. The situation is nebulous, however, and little is certain.
  </p>
  <p>
    You are Cpt. William "B.J." Blazkowicz, an Allied spy, a fearless risk-taker, and the greatest soldier to ever take up arms in World War II. Your role has changed, though; you've retired from active duty, and now you spearhead operations as a systems analyst. You thought it would be the right choice--leading and motivating instead of doing the dirty work--but lately you've grown restless. At least until a few days ago, when you received an encrypted message from your old friend and comrade Cpt. Douglas Blake, calling you back to duty...
  </p>
  <p>
    FEATURES

    Blade of Agony is much more than just a pastiche of the classic shooters that inspired it; the project will offer a unique experience. It also has many features not often seen in source port projects--Blade of Agony truly pushes GZDoom to its limits.
  </p>
  <ul>
    <li>17 playable and unique levels in Chapter 1 &amp; 2 (over 24 in the full game)</li>
    <li>Orchestra-quality game music</li>
    <li>Voice acting and ambient soundscapes</li>
    <li>A combination of low-poly models and hi-resolution sprite assets to create the perfect retro feeling</li>
    <li>Devastating armaments from the battlefields of WWII</li>
    <li>Interactive NPCs that support the twisted plot's progress and help you understand the game</li>
    <li>Beautiful special effects (weather, elements, explosions, etc.)</li>
    <li>Much, much more!</li>
  </ul>
  </description>
  <screenshots>
    <screenshot type="default">
      <image>http://boa.realm667.com/images/screenshots/c1/Screenshot_Doom_20160626_153131.jpg</image>
      <caption>CxMx: A description</caption>
    </screenshot>
    <screenshot>
      <image>http://boa.realm667.com/images/screenshots/c1/Screenshot_Doom_20160626_153236.jpg</image>
      <caption>CxMx: A description</caption>
    </screenshot>
    <screenshot>
      <image>http://boa.realm667.com/images/screenshots/c1/Screenshot_Doom_20160626_153412.jpg</image>
      <caption>CxMx: A description</caption>
    </screenshot>
    <screenshot>
      <image>http://boa.realm667.com/images/screenshots/c1/Screenshot_Doom_20160626_153609.jpg</image>
      <caption>CxMx: A description</caption>
    </screenshot>
  </screenshots>
</component>
```

**Comment by Tormentor667** [comment]
Feel free to make a pull request

**Comment by Alexander-Wilms** [comment]
Done: https://github.com/Realm667/WolfenDoom/pull/144

**Comment by Talon1024** [comment]
Fixed in #144
